README.md

```diff
@@ -62,17 +69,20 @@ app
 The client side code to support this is very simple too:
 
 ```html
-<script src="node_modules/koa-web-monetization/client.js"></script>
+<script src="/scripts/client.js"></script>
 <script>
-  getMonetizationId('http://localhost:8080/pay/:id')
-    .then(id => {
-      var img = document.createElement('img')
-      var container = document.getElementById('container')
-
-      img.src = '/content/' + id
-      img.width = '600'
-      container.appendChild(img)
-    })
+  var monetizerClient = new MonetizerClient();
+  monetizerClient.start()
+  .then(function() {
+    var img = document.createElement('img')
```

Comment:
If awaitBalance is used in the route for `content`, shouldn't you be able to just have the image as a part of the page instead of loading it after calling monetize?

Reply:
In this implementation awaitBalance is put into ctx.state after the client pings its receiverEndPointUrl, if the image tries to fetch content before that then the server freezes.